app/src/main/java/com/h/pixeldroid/objects/Account.kt

```diff
@@ -23,7 +23,7 @@ data class Account(
     val acct: String = "",
     val url: String = "", //HTTPS URL
     //Display attributes
-    val display_name: String? = null,
+    val display_name: String = "",
```

Comment:
Now that display_name is not nullable, you should just change `getUsername()` in Status.kt (line 125).